server/.env.example

```
@@ -14,4 +14,5 @@ CLDN_NAME=[nombre_de_ambiente_cloudinary_aqui]
 CLDN_API_KEY=[api_key_cloudinary_aqui]
 CLDN_SECRET_KEY=[secret_key_cloudinary_aqui]
 STRIPE_SECRET_KEY=[clave_secreta_stripe_aqui]
-STRIPE_WEBHOOK_SECRET=[clave_webhook_stripe_aqui]
\ No newline at end of file
+STRIPE_WEBHOOK_SECRET=[clave_webhook_stripe_aqui]
+ML_SERVICE_URL=[url_del_microservicio_ml_aqui]
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)